dist/cms.js

```diff
@@ -151,12 +151,31 @@ function getWordPressContentType() {
   return content;
 }
 
+/**
+ * Obtains data about Interactivity API elements on the page..
+ *
+ * @returns {object} Object with fields `usesInteractivityAPI`, `interactiveElementsCount`, and `interactiveElements`.
+ */
+function getInteractivityAPIData() {
+  // Look for data-wp-interactive elements.
+  const interactiveElements = document.querySelectorAll('[data-wp-interactive]');
```

Comment:
This gets not all interactive elements, but all interactive "regions".

For interactive elements, we would need to get all elements that use any of the Interactivity API's data attributes (like `data-wp-on-...`, `data-wp-bind-...`, `data-wp-class-...`, `data-wp-init-...`, `data-wp-watch-...` etc.).

Reply:
> This gets not all interactive elements, but all interactive "regions".

ok, looking at - https://developer.wordpress.org/block-editor/reference-guides/packages/packages-interactivity/packages-interactivity-api-reference/#wp-interactive I guess this is still a required wrapper so would be good to use for the boolean hasInetractivityAPI

> For interactive elements, we would need to get all elements that use any of the Interactivity API's data attributes (like data-wp-on-..., data-wp-bind-..., data-wp-class-..., data-wp-init-..., data-wp-watch-... etc.).

Ok, I will catalogue all of these are and include their data here broken out for each one, starting with counts for each type.

Reply:
I updated the keys so it is clear this is counting regions not elements. Looking at the other attributes that might be present seems less important, I think the high level count of usage is more relevant for scaled analysis.